tests/boulder-start.sh

```diff
@@ -38,5 +38,5 @@ if ! go get bitbucket.org/liamstask/goose/cmd/goose ; then
   exit 1
 fi
 ./test/create_db.sh
-./start.py &
+./start.py > /dev/null &
```

Comment:
I'm not sure about this one - it will make both Boulder and client debugging a lot harder (client doesn't log all actions - #912 is only one of the examples).